modules/sharedaddy/sharedaddy.php

```diff
@@ -68,7 +68,14 @@ function sharing_email_send_post_content( $data ) {
 	$content  = sprintf( __( '%1$s (%2$s) thinks you may be interested in the following post:', 'jetpack' ), $data['name'], $data['source'] );
 	$content .= "\n\n";
 	$content .= $data['post']->post_title."\n";
-	$content .= get_permalink( $data['post']->ID )."\n";
+	
+	// Filtered share URL. Allows backwards compatibility.
+	if ( isset( $data['share_url'] ) && ! empty( $data['share_url'] ) ) {
+		$content .= $data['share_url'] . "\n";
+	} else {
+		$content .= get_permalink( $data['post']->ID )."\n";
```

Comment:
Spaces around the concatenation dot, please?  I know it didn't have them previously, but since we're modifying the line's blame history anyways, let's fix it.